### DISCUSSION TITLE
Deploy using Now API

### DISCUSSION THREAD

**Discussion by elie222** [vercel/vercel#4197] [General] [answered]
We generate new GitHub repos from a template using the GitHub API and we'd like to autodeploy those to Now.

I see a lot of the old functionality that Zeit provided is deprecated in favour of using `npm link`. Is there a way to automatically connect and deploy a GitHub repo to now without first doing some steps in the UI?

**Comment by paulogdm** [comment]
Sure, but you would need our API to do that successfully: https://vercel.com/docs/api#endpoints/projects

**Comment by elie222** [comment]
I have the read the API documentation. Where does it show me how to connect to GitHub? I want to take a GitHub repo and create and a deploy to Zeit from it.

**Reply by paulogdm** [reply]
Oh, I see. Well, I don't think that is possible currently given our implementation. My idea was that you could manage the project creation and deployment via API.

**Reply by elie222** [reply]
Thanks. I figured out how to do this using Netlify last night instead but would have been great to do it with Vercel.

**Reply by paulogdm** [reply]
For sure. I have relayed to the team.

**Comment by elie222** [comment, marked as answer]
Closing this. Not currently possible.
